content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -167,9 +167,13 @@ The finalizer will only be removed after the load balancer resource is cleaned u
 This prevents dangling load balancer resources even in corner cases such as the
 service controller crashing.
 
-This feature is beta and enabled by default since Kubernetes v1.16. You can also
-enable it in v1.15 (alpha) via the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-`ServiceLoadBalancerFinalizer`.
+### Feature availablity
```

Comment:
I recommend using a [feature state shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state) to indicate support. For example, add that shortcode on line 157 and move the text down.

One you've done that, the table of feature support here isn't needed.

If you mark this GA feature as stable since v1.17, readers who run (eg) v1.16 should have enough information to either switch to reading the v1.16 docs or to check the feature gate list.

Reply:
👍

Reply:
Thanks @sftim, that's a great suggestion. Updated to use feature state shortcode instead.